### ISSUE TITLE
500. That’s an error. !!!!!HELP!!!!!

### ISSUE THREAD

**Issue by parajbs** [NastuzziSamy/files_external_gdrive#7]
hi Guys
my nextcloud version 13.0.1
and
files_external_gdrive 0.23 - 0.27 tested, 
but there is an error message in the browser (Chrome / Firefox / Opera):

500. That's an error.

There was an error. Please try again later. That's all we know.

and in the Nextcloud log:
----------------------

Error files_external Exception: Google storage not yet configured

/var/www/html/apps/files_external/lib/config.php - line 231: OCA \ Files_external_gdrive \ Storage \ GoogleDrive -> __ construct (Array)

/var/www/html/apps/files_external/lib/Controller/StoragesController.php - line 256: OC_Mount_Config getBackendStatus ('\\ OCA \\ Files_exte ...', array, false, true)

/var/www/html/apps/files_external/lib/Controller/UserStoragesController.php - line 207: OCA \ Files_External \ Controller \ StoragesController-> updateStorageStatus (Object (OCA \ Files_External \ Lib \ StorageConfig), true)

[internal function] OCA \ Files_External \ Controller \ UserStoragesController-> update (33, 'GoogleDrive', 'files_external _...', 'oauth2 oauth2', Array, Array, true)

/var/www/html/lib/private/AppFramework/Http/Dispatcher.php - line 161: call_user_func_array (Array, Array)

/var/www/html/lib/private/AppFramework/Http/Dispatcher.php - line 91: OC \ AppFramework \ Http \ Dispatcher-> executeController (Object (OCA \ Files_External \ Controller \ UserStoragesController), 'update')

/var/www/html/lib/private/AppFramework/App.php - line 115: OC \ AppFramework \ Http \ Dispatcher-> dispatch (Object (OCA \ Files_External \ Controller \ UserStoragesController), 'update')

/var/www/html/lib/private/AppFramework/Routing/RouteActionHandler.php - line 47: OC \ AppFramework \ App main ('OCA \\ Files_Exter ...', 'update', Object (OC \ AppFramework \ DependencyInjection \ DIContainer), Array)

[internal function] OC \ AppFramework \ Routing \ RouteActionHandler -> __ invoke (Array)
/var/www/html/lib/private/Route/Router.php - line 297: call_user_func (Object (OC \ AppFramework \ Routing \ RouteActionHandler), array)

/var/www/html/lib/base.php - line 998: OC \ Route \ Router -> match ('/ apps / files_ext ...')

/var/www/html/index.php - line 37: OC handleRequest ()
{Main}

-------------------------------

What is wrong there?????????????????????

Have also checked the permissions!

I've successfully run Dropbox with me! 
So I've uninstalled it, but it always comes with the same ERROR 500!

-----------------------------

Which Authorized Redirect URIs must have ????

https://example.com/index.php/settings/personal/
https://example.com/index.php/personal/
https://example.com/index.php/settings/admin/externalstorages
https://example.com/settings/admin/externalstorages

------------------

Can someone help, because I'm starting to run out of ideas, what else can I do?

Or!!!

Is the app still broken?

Here are the instructions for which I have proceeded!
https://docs.nextcloud.com/server/13/admin_manual/configuration_files/external_storage/google.html

Oh, so !!! App I had to manually copy custom apps from apps to apps!

HELP!

**Comment by NastuzziSamy** [comment]
Hi, Where does the issue come from ? When you try to grant access and you are redirected to Google ?

**Comment by joekerna** [comment]
I've had the same error with version 2.8, when I'm logged into my google account already.
When I log out of google and then log back in with the authorization process, the error does not show.

**Comment by parajbs** [comment]
Hello everyone
Yes, I was already logged in to Chrome automatically!
Then, the error 500!
First thought logging out to Nextcloud!
Later also Chrome logged out, but it came the error 500!
Then I went to another PC and then tried it from there via Firefox (here I am not logged in to Goggle !!!)!
It works there then!
GoogleDrive in Nextcloud connected / In the Nextcloud Files is GoggleDrive watch and open also, where then see all the files watch, but there are new problems!

Since bug 500 has been fixed, issue Issues!
For the new problem I open a new issue!
